starlight/templates/views/home.html

```diff
@@ -19,13 +19,13 @@ <h1 class="ui black header">Filter by team: </h1>
     {% csrf_token %}
     <div class="field">
         <select class="ui selection dropdown" name="name" id="name">
-              <option value=None>-------</option>
+              <option value="empty" selected>--------</option>
```

Comment:
If you just have an empty string for the value here, does it work?

Reply:
No, when I set value="", the option does not show up in the dropdown menu